### ISSUE TITLE
Syntax: make ternary and if statement more similar

### ISSUE THREAD

**Issue by theqp** [ethereum/fe#597]
### What is wrong?
I think ternary looks way too different from `if` statement compared to languages like Kotlin, Scala, ReScript, Rust

Ternary currently works like this as specified in https://github.com/ethereum/fe/issues/130
```
1 if input > 5 else 0
```
but `if` works like this:
```
if input > 5:
  return 1
else:
  return 0
```

### How can it be fixed

I would propose (or something similar):

inspired by `if` statement:
```
if input > 5: 1 else: 0
```
inspired by Scala 3 & Haskell
```
if input > 5 then 1 else 0
```
inspired by Lisp (aka. just make it look like a call)
```
if(input > 5, 1, 0)
if(input > 5, then=1, else=0)
```

Advantages:
- easier to learn, remember and refactor

**Comment by sbillig** [comment]
Yeah, I agree that something like `if x: y else: z` would be better. We've also discussed making the `if` statement an expression as in rust; eg:
```
let w = if x:
  y
else:
  z
```
which would make the ternary just a normal `if` expr, written on a single line. Or, we could switch to a rustier syntax and do `if x { y } else { z }` etc.

**Comment by sbillig** [comment]
Defunct. We now use rust-style if expressions.
